Resources/Locale/en-US/_NF/accent/goblin_accent.ftl

```diff
@@ -392,32 +392,44 @@ accent-goblin-words-387 = through
 accent-goblin-words-388 = isn’t it?
 accent-goblin-words-389 = just
 accent-goblin-words-390 = knows
```

Comment:
You seem to have removed 391, but it's still listed in the replacement table.

Reply:
phooq

Reply:
fixed